.bazelci/presubmit.yml

```diff
@@ -82,6 +82,8 @@ platforms:
     - "-//tests/docker:deb_image_with_dpkgs"
     - "-//testdata:pkg_control_xz.deb"
     - "-//testdata:pkg_control_xz"
+    # Disabled as docker is not available
```

Comment:
Add TODO to enable once docker is supported on buildkite

Reply:
Done